go/memcached-operator/.gitignore

```diff
@@ -1,5 +1,5 @@
 #IDEA
-.idea/*
+.idea/workspace.xml
```

Comment:
The entire dir should be ignored, like in the top level `.gitignore`.

Reply:
done